pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -98,6 +99,10 @@ func getDefaultDefaultPreemptionArgs() *config.DefaultPreemptionArgs {
 	return dpa
 }
 
+var nodeResourcesFitFunc = func(plArgs apiruntime.Object, fh framework.Handle) (framework.Plugin, error) {
+	return noderesources.NewFit(plArgs, fh, feature.Features{EnablePodOverhead: true})
```

Comment:
does any test fail with the feature set to false? If not, maybe we can just leave everything empty to not give the impression that we cared. Although.... PodOverhead is already enabled by default.

Reply:
Tested, and no other tests fail without it so I removed the PodOverhead parameter... but since the function still needs to be wrapped, I kept the declaration just for simplicity. Does that sound ok?

Reply:
perfect!